doc/spec/api.md

```diff
@@ -130,19 +130,20 @@ specification to correspond with the versions enumerated here.
 
 This section covers client flows and details of the API endpoints. The URI
 layout of the new API is structured to support a rich authentication and
-authorization model by leveraging namespaces. All endpoints will be prefixed
-by the API version and the repository name:
+authorization model by leveraging namespaces. All endpoints will begin with an
+optional `<prefix>` (see [the http configuration](../configuration.md#http)),
+the API version and the repository name:
 
-    /v2/<name>/
+    <prefix>/v2/<name>/
 
 For example, an API endpoint that will work with the `library/ubuntu`
-repository, the URI prefix will be:
+repository, the URI will begin with:
 
-    /v2/library/ubuntu/
+    <prefix>/v2/library/ubuntu/
 
 This scheme provides rich access control over various operations and methods
-using the URI prefix and http methods that can be controlled in variety of
-ways.
+using the URI beginning and HTTP methods that can be controlled in variety of
+ways. The remainder of this document will treat the `<prefix>` as an empty string.
```

Comment:
I added mention of the `<prefix>` configuration here, as it was the first question I had while reading this part.